packages/react-scripts/scripts/start.js

```diff
@@ -143,7 +143,7 @@ checkBrowsers(paths.appPath, isInteractive)
       }
 
       console.log(chalk.cyan('Starting the development server...\n'));
-      openBrowser(urls.localUrlForBrowser);
+      if (!process.env.RS_PREVENT_OPEN) openBrowser(urls.localUrlForBrowser);
```

Comment:
it can be very annoying when a new tab is opened on start when you already have one opened, I prefer to open it manually

Reply:
You can already disable this behavior via `BROWSER=none`.
```suggestion
      openBrowser(urls.localUrlForBrowser);
```

Reply:
Thanks, fixed